docs/devguide/README.md

```diff
@@ -0,0 +1,84 @@
+# Developing JDK Mission Control
+This document explains how to get started developing JDK Mission Control in Eclipse.
+
+## Getting Eclipse
+First of all you should download the latest version of Eclipse. JMC is an RCP application, and thus, the easiest way to make changes to it, is to use the Eclipse IDE.
+
+There are various Eclipse bundles out there. Get (at least) the Eclipse IDE for Eclipse Committers. It adds some useful things, like PDE (the Plugin Development Environment), Git, the Marketplace client and more. You can also use the Eclipse IDE for Java Enterprise Developers.
+
+**You will need an Eclipse 2020-03 or later!**
+
+To get to the screen where you can select another packaging than the standard, click on the [Download Packages](https://www.eclipse.org/downloads/eclipse-packages) link on the Eclipse download page.
+
+Install it, start it and create a new workspace for your JMC work. Creating a new workspace is as easy as picking a new name when starting up your Eclipse in the dialog asking for a directory for the workspace:
+
+![Workspace Selection](images/workspace.png)
```

Comment:
[nit] Pre-fixing image name like 01-workspace.png , 02-setinstalledjre.png... 10-formattersettings.png . would help a bit if the user wants to view these settings in order in there Folder (Explorer / Finder).

Reply:
I'd like to keep it as it is. Partly because I'm lazy, but also because if we extend or change the devguide, we'd have to potentially change the names of a lot of files and the README in a corresponding number of places.